### ISSUE TITLE
Laravel is throwing an arror after adding 'Contentful\Laravel\ContentfulServiceProvider' to the providers array

### ISSUE THREAD

**Issue by maxtherocket** [contentful/contentful-laravel#1]
After running:

```
php composer.phar require contentful/contentful-bundle:@beta
```

And adding 'Contentful\Laravel\ContentfulServiceProvider' to the providers array in config/app.php, I get the following error:

FatalThrowableError in ProviderRepository.php line 146:
Class 'Contentful\Laravel\ContentfulServiceProvider' not found

PHP 7.0.8
Laravel Framework version 5.3.6

**Comment by tommymarshall** [comment]
Sill happening.

**Comment by jevets** [comment]
Install `contentful/laravel` instead of `contentful/contentful-bundle`

`php composer require contentful/laravel`

You'll also need to set in your composer.json file (see  #2):

```json
{
    "prefer-stable": true,
    "minimum-stability": "dev"
}
```

**Comment by realityking** [comment]
Fixed in #2 thanks to @jevets.
